examples/arm/raspberry_pi/pico2/README.md

```diff
@@ -0,0 +1,86 @@
+## Overview
```

Comment:
Too much code in the readme for my taste. I would write a script and ask users to use that. If you want we can put that script in CI as well (until PTE generation and runner bin building)